auth0_flutter/example/ios/Tests/WebAuth/WebAuthLoginMethodHandlerTests.swift

```diff
@@ -174,6 +174,18 @@ extension WebAuthLoginHandlerTests {
         sut.handle(with: arguments(without: Argument.maxAge)) { _ in }
         XCTAssertNil(spy.maxAgeValue)
     }
+    
+    // MARK: safariViewController
+    
+    func testAddsSafariViewController() {
+        sut.handle(with: arguments(withKey: SafariViewController.key, value: [String: Any]())) { _ in }
+        XCTAssertNotNil(spy.provider)
+    }
+    
+    func testDoesNotAddSafariViewController() {
+        sut.handle(with: arguments(without: SafariViewController.key)) { _ in }
```

Comment:
Consider adding an entry for this argument in the `arguments` dictionary at the end of the file, with `[:]` as the value.

Reply:
Then, in the first test you can simply use `arguments()` because you're not matching a specific value –unlike other arg tests.

Reply:
I'm not _exactly_ sure what this is doing, but I can see the benefit - thanks

Reply:
The `arguments` dict allows you to set up a mock for all arguments. Then in your unit tests you can simply omit one or set a specific value, and assert. In this case it's a bit different, as mentioned, because we're not asserting a specific value.